symfony/src/Service/DeceasedReportsService.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace App\Service;
+
+use App\Service\RdrApiService;
+use App\Entity\DeceasedReport;
+use Pmi\Security\User;
+
+class DeceasedReportsService
+{
+    protected $api;
+
+    public function __construct(RdrApiService $api)
+    {
+        $this->api = $api;
+    }
+
+    public function getDeceasedReports($organizationId = null, $status = null)
```

Comment:
Related to comment above - maybe `$organizationId` shouldn't be allowed to be null.

Reply:
I'm hesitant to do this as the service _might_ be used to grab all `DeceasedReports` for the email notification bit (depending on if we process those by Organization or if we collect them all the pending approval ones and then only get the organization details for the batch). If it's no longer optional, I'd want to rename the method to `getDeceasedReportsByOrganizationId()`